src/rpc/evo.cpp

```diff
@@ -997,8 +1015,8 @@ static UniValue protx_update_service_common_wrapper(const JSONRPCRequest& reques
             // use operator reward address as default source for fees
             ExtractDestination(ptx.scriptOperatorPayout, feeSource);
         } else {
-            // use payout address as default source for fees
-            ExtractDestination(dmn->pdmnState->scriptPayout, feeSource);
+            // use first payout address as default source for fees
+            ExtractDestination(dmn->pdmnState->payoutShares[0].scriptPayout, feeSource);
```

Comment:
Ok here the situation is a bit more tricky because the "source for fees" is the address that spends an `utxo` and receives back `utxo - (fees needed to fund the tx)` ... So if we want to replicate we would have to put all payees address also as inputs... and we would also check that all those addresses belongs to the wallet + each address has has at least 1 utxo to fund the tx.

At the same time I agree that the first address as nothing special compared to the others... so I suggest that we try to fund the tx with payees addresses only if there is only 1 payee,  what do you think?

Reply:
if there is more than 1 payee transaction can be still funded with operatorAddress or the address specified by the user in the rpc